dnf/automatic/main.py

```diff
@@ -168,6 +168,9 @@ def main(args):
             base.do_transaction()
             emitters.notify_applied()
             emitters.commit()
+    except dnf.exceptions.ExitOnLock as exc:
+        logger.info(ucd(exc))
+        return 0
```

Comment:
I suppose that 1 would be better here. Please take into account also YUM behavior.